Lib/tempfile.py

```diff
@@ -43,12 +43,9 @@
 import shutil as _shutil
 import errno as _errno
 from random import Random as _Random
+import sys as _sys
 import weakref as _weakref
-
-try:
-    import _thread
-except ImportError:
-    import _dummy_thread as _thread
+import _thread
```

Comment:
Could you keep this in? We patched it to allow running without `--features threading`

Reply:
It was in there from CPython 3.6, but it got removed in response to [this issue on bugs.python.org](https://bugs.python.org/issue31370).

Reply:
Yeah, we're probably going to keep threadless as an option for the time being, just since we don't have a GIL option and can't enable threading without taking a performance hit everywhere. Plus we target wasm which doesn't have threads.